src/Apps/Legion/Hooks/useEntities.tsx

```diff
@@ -89,7 +89,7 @@ export const useEntities = () => {
                 deleted: 0
             };
             wizardDataState.entities.forEach((x) => {
-                if (x.typeId === typeId) {
+                if (typeId !== undefined && x.typeId === typeId) {
```

Comment:
it's probably safer to use the `isDefined` helper since it will also check for null. OR just do `!!typeId`

Reply:
Originally I had modified this function to take an optional typeId so that if the type was undefined we get total counts not scoped to a type. I need to add that logic back since we'll use this function for total new types. It'll be something like this:
if (!isDefined(typeId) || (isDefined(typeId) && x.typeId === typeId))